src/app/lib/prisma.ts

```diff
@@ -21,12 +21,6 @@ prisma.$use(async (params, next) => {
   const result = await next(params);
   const after = Date.now();
 
-  console.log(
```

Comment:
Hey! Sorry let's add this one back in. We are using it a lot for testing purposes right now as we move to Rest + Prisma.